test/applytests.jl

```diff
@@ -58,7 +58,11 @@ import ArrayLayouts: StridedLayout
         @test copyto!(b, a) == a == b
         @test copyto!(c, a) == a == c
         @test copyto!(similar(a), a) == a
-        @test_throws ErrorException copyto!(c, Array(a))
+        if VERSION ≤ v"1.7.99"
```

Comment:
```suggestion
        if VERSION ≤ v"1.8.0-DEV"
```

Reply:
This would also work:

```
if VERSION < v"1.8.0-DEV"
```

Reply:
Great! I knew there was a better way. Thanks